app/src/main/java/ch/epfl/sdp/game/Server.java

```diff
@@ -218,6 +230,7 @@ private void addPlayersPositionListener() {
 
                         // update the location of the player
                         player.setLocation(location);
+                        player.setAoeRadius(playerForFirebase.getAoeRadius());
```

Comment:
For now, do we have the method to send the aoeRadius to cloud firebase on Client side?

Reply:
For now it is not updated on the database.

Reply:
Has been changed: we have a sendAoeRadius method.